src/styles/blocks/section3.scss

```diff
@@ -0,0 +1,31 @@
+.section_3 {
+  display: flex;
+  height: 40px;
+  width: 166px;
+  justify-content: stretch;
+}
+
+.section_3__link {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 12px 68px;
+  height: 40px;
+  background-color: #00acdc;
+  flex-grow: 1;
+  text-align: center;
+  justify-content: center;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  color: white;
+  text-decoration: none;
+  width: 100%;
```

Comment:
Setting the width to 100% may not be necessary here if you want the link to grow and fill the space of its flex container. The 'flex-grow: 1' property in line 14 will already ensure that it takes up the available space.